src/Transports.AspNetCore/IUserContextBuilder.cs

```diff
@@ -1,16 +1,17 @@
 using Microsoft.AspNetCore.Http;
 
-namespace GraphQL.Server.Transports.AspNetCore;
-
-/// <summary>
-/// Interface which is responsible of building a UserContext for a GraphQL request
-/// </summary>
-public interface IUserContextBuilder
+namespace GraphQL.Server.Transports.AspNetCore
```

Comment:
Not file-scoped. Merge artefact 🤔?

Reply:
Probably; I can fix easy enough.

Reply:
This entire file had not been updated to my copy.  I have done so; see updated comments.